tests/python/contrib/test_hexagon/test_cache_read_write.py

```diff
@@ -222,5 +223,28 @@ def _visit(stmt):
     )
 
 
+def _raises_exception(f):
+    try:
+        f()
+    except Exception:
+        return True
+    return False
+
+
+@pytest.mark.parametrize("vtcm_capacity,limited", [(65536, False), (8192, True)])
+def test_vtcm_limit(vtcm_capacity, limited):
+    """Test lowering with vtcm mem scope"""
+    mod = tvm.IRModule.from_expr(scale_by_two.with_attr("global_symbol", "main"))
+    sch = tir.Schedule(mod, debug_mask="all")
+    block_c = sch.get_block("C")
+    (flat,) = sch.get_loops(block_c)
+    outer, _, _, _ = sch.split(flat, factors=[8, 4, 2, 128])
+    cache_block = sch.cache_read(block_c, 0, storage_scope="global.vtcm")
+    sch.compute_at(cache_block, outer)
+
+    with tvm.transform.PassContext(config={"tir.vtcm_capacity": vtcm_capacity}):
```

Comment:
It would be interesting in the future if we could pass this capacity down to the device API for two reasons - to verify that it is possible, and to attempt to allocate only that size from QURT in HexagonVtcmPool